internal/providers/packet/packet.go

```diff
@@ -48,3 +61,85 @@ func FetchConfig(f resource.Fetcher) (types.Config, report.Report, error) {
 
 	return util.ParseConfig(f.Logger, data)
 }
+
+// PostStatus posts a message that will show on the Packet Instance Timeline
+func PostStatus(f resource.Fetcher, errMsg error) error {
+	f.Logger.Info("POST message to Packet Timeline")
+	// fetch JSON from https://metadata.packet.net/metadata
+	data, err := f.FetchToBuffer(metadataUrl, resource.FetchOptions{
+		Headers: nil,
+	})
+	if err != nil {
+		return err
+	}
+	if data == nil {
+		return errors.ErrValidFetchEmptyData
+	}
+
+	metadata := struct {
+		PhoneHomeURL string `json:"phone_home_url"`
+	}{}
+	err = json.Unmarshal(data, &metadata)
+	if err != nil {
+		return err
+
+	}
+	phonehomeURL := metadata.PhoneHomeURL
+	// to get phonehome IPv4
+	phonehomeURL = strings.TrimSuffix(phonehomeURL, "/phone-home")
+	// POST Message to phonehome IP
+	postMessageURL := phonehomeURL + "/events"
+	if errMsg != nil {
+
+		err = postMessage(errMsg, f, "Ignition error: "+errMsg.Error(), postMessageURL)
+		if err != nil {
+			return err
+		}
+
+	} else {
+
+		err = postMessage(errMsg, f, "Ignition: valid config", postMessageURL)
```

Comment:
s/Ignition: valid config/Ignition stage <stage> exited successful
You can have a valid config and still fail.